src/components/BlockNativeOnboard/index.tsx

```diff
@@ -81,7 +81,7 @@ export const OnboardProviderWrapper = ({ children }) => {
     return (
         <OnboardProvider
             options={connectOptions}
-            wallets={{ custom: [torus] }}
+            // wallets={{ custom: [torus] }}
```

Comment:
why?

Reply:
Forgot to look into and fix. done here fixing a dep conflict: https://github.com/GoodDollar/GoodProtocolUI/pull/578/commits/77ec17fc8f7921d3dda575c14b729cfdda25d327

Reply:
wasnt that patch required for the custom wallet icons or something?